src/adapters/conversant.js

```diff
@@ -0,0 +1,230 @@
+        imp;
+
+      secure = utils.getBidIdParamater('secure', bid.params) ? 1 : secure;
+      siteId = utils.getBidIdParamater('site_id', bid.params);
+
+      if (sizeArrayLength === 2 && typeof bid.sizes[0] === 'number' && typeof bid.sizes[1] === 'number') {
+        adW = bid.sizes[0];
+        adH = bid.sizes[1];
+      } else {
+        adW = bid.sizes[0][0];
+        adH = bid.sizes[0][1];
+      }
+
+      imp = {
+        id: bid.bidId,
+        banner: {
+          w: adW,
+          h: adH
+        },
+        secure: secure,
+        bidfloor: bidfloor ? bidfloor : 0,
+        displaymanager: 'Prebid.js',
+        displaymanagerver: VERSION
+      };
+
+      conversantImps.push(imp);
+    });
+
+    conversantBidReqs = {
+      'id': utils.getUniqueIdentifierStr(),
+      'imp': conversantImps,
+
+      'site': {
+        'id': siteId,
+        'mobile': document.querySelector('meta[name="viewport"][content*="width=device-width"]') !== null ? 1 : 0,
+        'page': page
+      },
+
+      'device': getDevice(),
+      'at': 1
+    };
+
+    var url = secure ? 'https:' + conversantUrl : location.protocol + conversantUrl;
+    httpPOSTAsync(url, JSON.stringify(conversantBidReqs));
+  };
+
+  var addEmptyBidResponses = function (placementsWithBidsBack) {
+    var allConversantBidRequests = $$PREBID_GLOBAL$$._bidsRequested.find(bidSet => bidSet.bidderCode === 'conversant');
+
+    if (allConversantBidRequests && allConversantBidRequests.bids){
+      utils._each(allConversantBidRequests.bids, function (conversantBid) {
+        if (!utils.contains(placementsWithBidsBack, conversantBid.placementCode)) {
+          // Add a no-bid response for this placement.
+          var bid = bidfactory.createBid(2, conversantBid);
+          bid.bidderCode = 'conversant';
+          bidmanager.addBidResponse(conversantBid.placementCode, bid);
+        }
+      });
+    }
+  };
+
+  var parseSeatbid = function (bidResponse) {
+    var placementsWithBidsBack = [];
+    utils._each(bidResponse.bid, function (conversantBid) {
+      var responseCPM,
+        placementCode = '',
+        id = conversantBid.impid,
+        bid = {},
+        responseAd,
+        responseNurl,
+        sizeArrayLength;
+
+      // Bid request we sent Conversant
+      var bidRequested = $$PREBID_GLOBAL$$._bidsRequested.find(bidSet => bidSet.bidderCode === 'conversant').bids.find(bid => bid.bidId === id);
+
+      if (bidRequested) {
+        placementCode = bidRequested.placementCode;
+        placementsWithBidsBack.push(placementCode);
+        bidRequested.status = CONSTANTS.STATUS.GOOD;
+
+        responseCPM = parseFloat(conversantBid.price);
+
+        if (responseCPM !== 0.0) {
+          conversantBid.placementCode = placementCode;
+          conversantBid.size = bidRequested.sizes;
+          responseAd = conversantBid.adm || '';
+          responseNurl = conversantBid.nurl || '';
+
+          // Our bid!
+          bid = bidfactory.createBid(1);
```

Comment:
pass the bid object as 2nd argument to sync bid request and response IDs. See #509